nemo/collections/tts/waveglow_modules.py

```diff
@@ -146,27 +153,24 @@ def input_ports(self):
         """Returns definitions of module input ports.
         """
         return {
-            # "mel_spectrogram": NeuralType(
-            #     {0: AxisType(BatchTag), 1: AxisType(MelSpectrogramSignalTag), 2: AxisType(TimeTag),}
-            # )
-            "mel_spectrogram": NeuralType(('B', 'D', 'T'), MelSpectrogramType())
+            "mel_spectrogram": NeuralType(('B', 'D', 'T'), MelSpectrogramType()),
+            "z": NeuralType(('B', 'T'), AudioSignal(freq=self.sample_rate)),
         }
 
     @property
     @add_port_docs()
     def output_ports(self):
         """Returns definitions of module output ports.
         """
-        # return {"audio": NeuralType({0: AxisType(BatchTag), 1: AxisType(TimeTag)})}
         return {"audio": NeuralType(('B', 'T'), AudioSignal(freq=self.sample_rate))}
 
     def __str__(self):
-        return "WaveGlowNM"
+        return "WaveGlowInferNM"
 
     def __init__(
         self,
-        *,
-        sample_rate: int,
+        waveglow=None,
```

Comment:
this leaks backend (Pytorch) into NeMo module signature. We make an effort to avoid it. Also, this would break this example: examples/tts/waveglow.py. The most important part about that example is that it does not import torch.